include/modem/lte_lc.h

```diff
@@ -617,13 +632,24 @@ struct lte_lc_conn_eval_params {
 
 /** @brief Specifies which type of search the modem should perform when a neighbor
  *	   cell measurement is started.
+ *
+ *         When using search types up to LTE_LC_NEIGHBOR_SEARCH_TYPE_EXTENDED_COMPLETE,
+ *         result contains parameters from serving/current cell
+ *         and optionally up to CONFIG_LTE_NEIGHBOR_CELLS_MAX neighbor cells.
```

Comment:
> @peknis It would also be nice to have some input on how to refer to Kconfig options in header files

Not so sure about this either, but will find out soon.

Reply:
Here goes:

you cannot link to them from the header files because they are not part of the doxygen source